fpa/auth/repo.go

```diff
@@ -0,0 +1,77 @@
+package auth
```

Comment:
_fps/auth_ is a package that has the information about which _auth-tokens_ for a FastAPI instance maps with the FastAPI instance ID. In the future any kind of _auth-token_ invalidation, expiry, or things related should live here.
TL;DR: mapping between auth-tokens to FastAPI instances.